src/main/plugins/org.dita.pdf2/xsl/fo/troubleshooting-elements.xsl

```diff
@@ -0,0 +1,205 @@
+                <xsl:apply-templates select="." mode="dita2xslfo:retrieve-task-heading">
+                    <xsl:with-param name="pdf2-string">Troubleshooting Step</xsl:with-param>
+                    <xsl:with-param name="common-string">trouble_step</xsl:with-param>
+                </xsl:apply-templates>
+            </xsl:with-param>
+        </xsl:apply-templates>
+        <fo:block xsl:use-attribute-sets="steptroubleshooting">
+            <xsl:call-template name="commonattributes"/>
+            <xsl:apply-templates/>
+        </fo:block>
+    </xsl:template>
+
+    <xsl:template match="*[contains(@class, ' task/tasktroubleshooting ')]">
+        <fo:block xsl:use-attribute-sets="tasktroubleshooting">
+            <xsl:call-template name="commonattributes"/>
+            <xsl:apply-templates select="." mode="dita2xslfo:task-heading">
+                <xsl:with-param name="use-label">
+                    <xsl:apply-templates select="." mode="dita2xslfo:retrieve-task-heading">
+                        <xsl:with-param name="pdf2-string">Troubleshooting Task</xsl:with-param>
+                        <xsl:with-param name="common-string">trouble_task</xsl:with-param>
+                    </xsl:apply-templates>
+                </xsl:with-param>
+            </xsl:apply-templates>
+            <fo:block xsl:use-attribute-sets="tasktroubleshooting__content">
+              <xsl:apply-templates/>
+            </fo:block>
+        </fo:block>
+    </xsl:template>
+
+    <!-- If example has a title, process it first; otherwise, create default title (if needed) -->
+    <xsl:template match="*[contains(@class, ' troubleshooting/troublebody ')]/*[contains(@class, ' topic/example ')]">
+        <fo:block xsl:use-attribute-sets="task.example">
+            <xsl:call-template name="commonattributes"/>
+            <xsl:choose>
+              <xsl:when test="*[contains(@class, ' topic/title ')]">
+                <xsl:apply-templates select="*[contains(@class, ' topic/title ')]"/>
+              </xsl:when>
+              <xsl:otherwise>
+                <xsl:apply-templates select="." mode="dita2xslfo:task-heading">
+                  <xsl:with-param name="use-label">
+                      <xsl:apply-templates select="." mode="dita2xslfo:retrieve-task-heading">
+                        <xsl:with-param name="pdf2-string">Task Example</xsl:with-param>
+                        <xsl:with-param name="common-string">task_example</xsl:with-param>
+                      </xsl:apply-templates>
+                  </xsl:with-param>
+                </xsl:apply-templates>
+              </xsl:otherwise>
+            </xsl:choose>
+            <fo:block xsl:use-attribute-sets="task.example__content">
+              <xsl:apply-templates select="*[not(contains(@class, ' topic/title '))]|text()|processing-instruction()"/>
+            </fo:block>
+        </fo:block>
+    </xsl:template>
+
+    <xsl:template match="*" mode="dita2xslfo:task-heading">

```

Comment:
This should either be removed, or modified so that it is specific to troubleshooting headings. Currently, this template will override the template already in place for task elements.